app/pdfs/certificate_pdf.rb

```diff
@@ -0,0 +1,48 @@
+require 'open-uri'
+
+class CertificatePDF < Prawn::Document
+  attr_accessor :path
```

Comment:
CertificatePDF#path is a writable attribute

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Attribute.md)